### ISSUE TITLE
No checkpoints saved after validation epoch ends

### ISSUE THREAD

**Issue by lhatsk** [aqlaboratory/openfold#51]
Checkpoints are not saved after validation epoch ends. checkpoint_best_val is active

Validation loss is not shown during validation, maybe this is connected? (since it's supposed to track val_loss)

**Comment by gahdritz** [comment]
It looks like the PyTorch Lightning idiom's changed recently: https://pytorch-lightning.readthedocs.io/en/stable/common/weights_loading.html. They now seem to be using `self.log` rather than returning dictionaries. I'll test and fix that myself as soon as I get the chance. I just thought I'd flag that for you in the meantime.

**Comment by lhatsk** [comment]
Thanks! That solves it. I just added self.log("val_loss", loss), kept the return for now.

Validation loss is unfortunately still not shown in the progress bar.
